app/stories/modal.stories.jsx

```diff
@@ -382,6 +383,12 @@ story('Marketplace/InsufficientDaiDialog')
             onCancel={action('onCancel')}
         />
     ))
+story('Marketplace/InsufficientEthDialog')
+    .add('default', () => (
+        <InsufficientEthDialog
+            onCancel={action('onCancel')}
+        />
+    ))
```

Comment:
Could you add a mobile story for dialogs where there is some difference, eg.

```js
story('Marketplace/InsufficientEthDialog')
    .add('default', () => (
        <InsufficientEthDialog
            onCancel={action('onCancel')}
        />
    ), {
    viewport: {
        defaultViewport: 'sm',
    },
})
```

Reply:
Hmm, there's a wrinkle in this approach - `isMobile()` uses userAgent sniffing - it won't respond to the storybook viewporting. I will have to think about this one a bit more. 

![Screenshot 2020-02-12 at 12 00 33](https://user-images.githubusercontent.com/1593398/74329483-2df6cd80-4d90-11ea-9a87-850be4e54e0b.png)

Reply:
I think the easiest way is to print both texts and hide/show the correct one with css

Reply:
true 👍

Reply:
Done. I added a custom viewport 'iPhone' because these sort of line break issues only appear at that resolution.